web/assets/script.js

```diff
@@ -87,16 +87,21 @@ function render(data, counties)
             if(entry.incidence <= 35)
                 color = "green";
             else if(entry.incidence <= 50)
-                color = "orange";
+                color = "yellow";
             else if(entry.incidence <= 100)
-                color = "red";
+                color = "orange";
             else if(entry.incidence <= 200)
+                color = "red";
+            else if(entry.incidence <= 500)
                 color = "darkred";
             else if(entry.incidence <= 1000)
                 color = "purple";
+            else if(entry.incidence <= 2000)
+                color = "black";
+                fillOpacity = 0.5
             else
                 color = "black";
-                fillOpacity = 0.6
+                fillOpacity = 0.8
```

Comment:
the `fillOpacity = 0.5` statement is not part of the `else if` and thus results in the following else to be a syntax error.

Reply:
Fixed, thank you. I wasn't aware of this because i alwayse use brackets